__tests__/app.test.js

```diff
@@ -89,47 +90,97 @@ describe("GET /api", () => {
   });
 });
 
-describe('GET /api/articles/:article_id', () => {
-    test('returns an object', () => {
-        return request(app)
-        .get("/api/articles/1")
-        .expect(200)
-        .then((response) => {
-            const article = response.body.article
-          expect(article.constructor).toBe(Object)
-        });
-    })
-    test('returns an object with correct properties ', () => {
-        return request(app)
-        .get("/api/articles/1")
-        .expect(200)
-        .then((response) => {
-            const article = response.body.article
-            expect(article.hasOwnProperty('author')).toBe(true)
-            expect(article.hasOwnProperty('title')).toBe(true)
-            expect(article.hasOwnProperty('article_id')).toBe(true)
-            expect(article.hasOwnProperty('body')).toBe(true)
-            expect(article.hasOwnProperty('topic')).toBe(true)
-            expect(article.hasOwnProperty('created_at')).toBe(true)
-            expect(article.hasOwnProperty('votes')).toBe(true)
-            expect(article.hasOwnProperty('article_img_url')).toBe(true)
-        })
-    });
-    test('returns 404 when given an id number that does not exist', () => {
-        return request(app)
-        .get("/api/articles/500")
-        .expect(404)
-        .then((response) => {
-            expect(response.body.msg).toBe("not found")
-        })
-    });
+describe("GET /api/articles/:article_id", () => {
+  test("returns an object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article.constructor).toBe(Object);
+      });
+  });
+  test("returns an object with correct properties ", () => {
```

Comment:
I'd much rather see this test show me the correct article from the database. All articles have the right keys, you're asking for article 1. If the server gave me article 13 this would pass and as a user I would be baffled. I'd like you to change this.